python/openai/tests/conftest.py

```diff
@@ -31,8 +31,10 @@
 from fastapi.testclient import TestClient
 from tests.utils import OpenAIServer, setup_fastapi_app, setup_server
 
-
 ### TEST ENVIRONMENT SETUP ###
+LLMAPI_SETUP = os.environ.get("LLMAPI_SETUP", 0)
```

Comment:
Would be nice to clean up some of the `if/else` everywhere for llmapi vs trtllm stuff, but hopefully we can consolidate the supported backend implementations once its a bit more mature.